tests/examples/CMakeLists.txt

```diff
@@ -14,22 +14,44 @@ cmake_minimum_required(VERSION 3.0.0)
 # Get and update the LUA_PATH so the scripts can be executed without Awesome.
 execute_process(COMMAND lua -e print\(package.path\) OUTPUT_VARIABLE "LUA_PATH_")
 
```

Comment:
For the commit message: How about some more details. The commit message does not even say that this commit fixes running the doc tests without compiling. I'd like to know what exactly was broken and was fixed (and perhaps also that "running the doc tests without compiling" means running `cmake` in the `tests/examples` directory (does it mean that?))